### ISSUE TITLE
[BITV] 9.1.4.3/3.5 - "Details" section / "Versions" tab - Text information regarding versions such as file size does not meet the contrast requirements for text. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36955]
- [x] Fix

![61f5c06cc82a2ea9f50eb6f75cc6db2d](https://user-images.githubusercontent.com/1315170/222224073-7a2f21df-3feb-4756-8059-273de077ef1e.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6fe46b78da-v3-n5

</details>

- [x] 9.1.4.3/10.5 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6fe46b78da-v10-n5

**Comment by Pytal** [comment]
Fixed on `master` with versions tab in Vue
